--- a/Makefile
+++ b/Makefile
@@ -11,15 +11,18 @@
 GCIN_SO= gcin1.so gcin2.so
 
 OBJS=gcin.o eve.o util.o gcin-conf.o gcin-settings.o locale.o gcin-icon.o \
-     gcin-switch.o gcin-exec-script.o $(GCIN_SO) pho-play.o cache.o gtk_bug_fix.o \
+     gcin-switch.o gcin-exec-script.o pho-play.o cache.o gtk_bug_fix.o \
      $(gcin_pho_o) $(gcin_gtab_o) gcin-common.o phrase.o t2s-lookup.o gtab-use-count.o \
-     win-save-phrase.o unix-exec.o
+     win-save-phrase.o unix-exec.o \
+     intcode.o win-int.o win-message.o win-sym.o \
+     win-inmd-switch.o pinyin.o win-pho-near.o win-kbm.o \
+     anthy.o
 
 OBJS_TSLEARN=tslearn.o util.o gcin-conf.o pho-util.o tsin-util.o gcin-send.o pho-sym.o \
              table-update.o locale.o gcin-settings.o gcin-common.o gcin-icon.o pho-dbg.o
 OBJS_JUYIN_LEARN=juyin-learn.o locale.o util.o pho-util.o pho-sym.o \
                  gcin-settings.o gcin-conf.o table-update.o pinyin.o gcin-icon.o pho-dbg.o
-OBJS_sim2trad=sim2trad.o util.o gcin2.so locale.o gcin-conf.o gcin-icon.o
+OBJS_sim2trad=sim2trad.o util.o locale.o gcin-conf.o gcin-icon.o t2s-lookup.o
 OBJS_phod2a=phod2a.o pho-util.o gcin-conf.o pho-sym.o table-update.o pho-dbg.o locale.o \
              gcin-settings.o util.o
 OBJS_tsa2d32=tsa2d32.o gcin-send.o util.o pho-sym.o gcin-conf.o locale.o pho-lookup.o
@@ -126,7 +129,6 @@
 	if [ $(QT4_IM) = 'Y' ]; then $(MAKE) -C qt4-im; fi
 
 gcin:   $(OBJS) $(IMdkitLIB) $(OBJ_IMSRV)
-	LD_RUN_PATH=$(gcin_ld_run_path) \
 	$(CCLD) $(EXTRA_LDFLAGS) -o $@ $(OBJS) $(IMdkitLIB) $(OBJ_IMSRV) -lXtst $(LDFLAGS) -L/usr/X11R6/lib
 	rm -f core.* vgcore.*
 	ln -sf $@ $@.test
@@ -144,7 +146,6 @@
 	$(CCLD) -o $@ $(OBJS_JUYIN_LEARN) $(LDFLAGS)
 	rm -f core.*
 sim2trad:        $(OBJS_sim2trad)
-	LD_RUN_PATH=$(gcin_ld_run_path) \
 	$(CC) -o $@ $(OBJS_sim2trad) $(LDFLAGS)
 	rm -f core.*
 trad2sim:	sim2trad
@@ -247,14 +248,11 @@
 	install $(PROGS) $(bindir); \
 	cp gcin-nocur $(bindir)/gcin; \
 	rm -f $(bindir)/trad2sim; ln -sf sim2trad $(bindir)/trad2sim
-	install $(GCIN_SO) $(gcinlibdir)
 
 install:
 	install -d $(datadir)/icons
 	install -m 644 gcin.png $(datadir)/icons
 	$(MAKE) -C icons install
-	install -d $(gcinlibdir)
-	install $(GCIN_SO) $(gcinlibdir)
 	install -d $(bindir)
 	$(MAKE) -C data install
 	$(MAKE) -C im-client install
