Fix 64bit builds
---
 configure |   15 ++++++---------
 1 file changed, 6 insertions(+), 9 deletions(-)

--- a/configure
+++ b/configure
@@ -145,9 +145,6 @@
 fi
 
 LIB='lib'
-if [ -d /lib64 ]; then
-LIB='lib64'
-fi
 
 QT_IM='N'
 QT_IM_DIR=/usr/$LIB/qt3/plugins/inputmethods
@@ -269,11 +266,11 @@
 echo "GCIN_SCRIPT_DIR=\$(datadir_r)/gcin/script" >> config.mak
 echo "GCIN_SCRIPT_DIR_i=\$(datadir)/gcin/script" >> config.mak
 echo "OPTFLAGS=$OPTFLAGS" >> config.mak
-echo "DOC_DIR=\$(datadir_r)/doc/gcin-$GCIN_VERSION" >> config.mak
-echo "DOC_DIR_i=\$(datadir)/doc/gcin-$GCIN_VERSION" >> config.mak
-echo "GCIN_ICON_DIR=\$(datadir_r)/icons/gcin" >> config.mak
-echo "SYS_ICON_DIR=\$(datadir_r)/icons" >> config.mak
-echo "GCIN_ICON_DIR_i=\$(datadir)/icons/gcin" >> config.mak
+echo "DOC_DIR=\$(datadir_r)/doc/gcin" >> config.mak
+echo "DOC_DIR_i=\$(datadir)/doc/gcin" >> config.mak
+echo "GCIN_ICON_DIR=\$(datadir_r)/pixmaps/gcin" >> config.mak
+echo "SYS_ICON_DIR=\$(datadir_r)/pixmaps" >> config.mak
+echo "GCIN_ICON_DIR_i=\$(datadir)/pixmaps/gcin" >> config.mak
 echo "GCIN_BIN_DIR=\$(bindir_r)" >> config.mak
 echo "FREEBSD=$FREEBSD" >> config.mak
 echo "QT_IM=$QT_IM" >> config.mak
