Fix rpath.
---
 Makefile.gcin |   17 +++++++----------
 1 file changed, 7 insertions(+), 10 deletions(-)

Index: gcin/Makefile.gcin
===================================================================
--- gcin.orig/Makefile.gcin	2008-12-08 22:01:37.114312956 +0800
+++ gcin/Makefile.gcin	2008-12-08 23:02:15.531313815 +0800
@@ -217,7 +217,7 @@
 	else \
 	   install -d $(DOC_DIR_i); \
 	   install -m 644 README.html Changelog.html $(DOC_DIR_i); \
-	   install -s $(PROGS) $(bindir); \
+	   install $(PROGS) $(bindir); \
 	   rm -f $(bindir)/trad2sim; ln -sf sim2trad $(bindir)/trad2sim; \
 	fi
 	$(MAKE) -C scripts install
