Fix 64bit builds
---
 configure |   15 ++++++---------
 1 file changed, 6 insertions(+), 9 deletions(-)

Index: gcin/configure
===================================================================
--- gcin.orig/configure	2008-12-08 16:53:10.669679144 +0800
+++ gcin/configure	2008-12-08 17:00:30.363679709 +0800
@@ -107,14 +107,11 @@
 fi
 
 LIB='lib'
-if [ -d /lib64 ]; then
-LIB='lib64'
-fi
 
 QT_IM='N'
 for i in qt3 qt4
 do
-  QT_IM_DIR=/usr/lib/$i/plugins/$LIB/inputmethods
+  QT_IM_DIR=/usr/lib/$i/plugins/inputmethods
   echo "testing if you have QT_IM_MODULE dir:$QT_IM_DIR support"
   if [ -d  $QT_IM_DIR ]; then
     QT_IM='Y'
@@ -184,11 +181,11 @@
 echo "GCIN_SCRIPT_DIR=\$(datadir_r)/gcin/script" >> config.mak
 echo "GCIN_SCRIPT_DIR_i=\$(datadir)/gcin/script" >> config.mak
 echo "OPTFLAGS=$OPTFLAGS" >> config.mak
-echo "DOC_DIR=\$(datadir_r)/doc/gcin-$GCIN_VERSION" >> config.mak
-echo "DOC_DIR_i=\$(datadir)/doc/gcin-$GCIN_VERSION" >> config.mak
-echo "GCIN_ICON_DIR=\$(datadir_r)/icons/gcin" >> config.mak
-echo "SYS_ICON_DIR=\$(datadir_r)/icons" >> config.mak
-echo "GCIN_ICON_DIR_i=\$(datadir)/icons/gcin" >> config.mak
+echo "DOC_DIR=\$(datadir_r)/doc/gcin" >> config.mak
+echo "DOC_DIR_i=\$(datadir)/doc/gcin" >> config.mak
+echo "GCIN_ICON_DIR=\$(datadir_r)/pixmaps/gcin" >> config.mak
+echo "SYS_ICON_DIR=\$(datadir_r)/pixmaps" >> config.mak
+echo "GCIN_ICON_DIR_i=\$(datadir)/pixmaps/gcin" >> config.mak
 echo "GCIN_BIN_DIR=\$(bindir_r)" >> config.mak
 echo "FREEBSD=$FREEBSD" >> config.mak
 echo "QT_IM=$QT_IM" >> config.mak
