Fix 64bit builds
---
 configure |   15 ++++++---------
 1 file changed, 6 insertions(+), 9 deletions(-)

Index: gcin/configure
===================================================================
--- gcin.orig/configure	2008-12-08 22:01:37.115313292 +0800
+++ gcin/configure	2008-12-08 22:40:59.228314107 +0800
@@ -107,12 +107,9 @@
 fi
 
 LIB='lib'
-if [ -d /lib64 ]; then
-LIB='lib64'
-fi
 
 QT_IM='N'
-QT_IM_DIR=/usr/lib/qt3/plugins/lib/inputmethods
+QT_IM_DIR=/usr/lib/qt3/plugins/inputmethods
 echo "testing if you have qt3 support"
 if [ -d  $QT_IM_DIR ]; then
   QT_IM='Y'
@@ -126,7 +123,7 @@
 
 
 QT4_IM='N'
-QT4_IM_DIR=/usr/lib/qt4/plugins/lib/inputmethods
+QT4_IM_DIR=/usr/lib/qt4/plugins/inputmethods
 echo "testing if you have qt4 support"
 if [ -d  $QT4_IM_DIR ]; then
   QT4_IM='Y'
@@ -199,11 +196,11 @@
 echo "GCIN_SCRIPT_DIR=\$(datadir_r)/gcin/script" >> config.mak
 echo "GCIN_SCRIPT_DIR_i=\$(datadir)/gcin/script" >> config.mak
 echo "OPTFLAGS=$OPTFLAGS" >> config.mak
-echo "DOC_DIR=\$(datadir_r)/doc/gcin-$GCIN_VERSION" >> config.mak
-echo "DOC_DIR_i=\$(datadir)/doc/gcin-$GCIN_VERSION" >> config.mak
-echo "GCIN_ICON_DIR=\$(datadir_r)/icons/gcin" >> config.mak
-echo "SYS_ICON_DIR=\$(datadir_r)/icons" >> config.mak
-echo "GCIN_ICON_DIR_i=\$(datadir)/icons/gcin" >> config.mak
+echo "DOC_DIR=\$(datadir_r)/doc/gcin" >> config.mak
+echo "DOC_DIR_i=\$(datadir)/doc/gcin" >> config.mak
+echo "GCIN_ICON_DIR=\$(datadir_r)/pixmaps/gcin" >> config.mak
+echo "SYS_ICON_DIR=\$(datadir_r)/pixmaps" >> config.mak
+echo "GCIN_ICON_DIR_i=\$(datadir)/pixmaps/gcin" >> config.mak
 echo "GCIN_BIN_DIR=\$(bindir_r)" >> config.mak
 echo "FREEBSD=$FREEBSD" >> config.mak
 echo "QT_IM=$QT_IM" >> config.mak
