Index: gcin/configure
===================================================================
--- gcin.orig/configure	2008-12-06 14:28:01.589493548 +0800
+++ gcin/configure	2008-12-06 14:28:01.667493782 +0800
@@ -155,9 +155,10 @@
 
 OPTFLAGS="-g"
 if [ $prefix = /usr ]; then
-  OPTFLAGS="-O"
+  OPTFLAGS="-g -O2"
 fi
 
+LDFLAGS="-Wl,--as-needed -lX11 $GTKLDFLAGS"
 
 echo "CC=gcc" > config.mak
 echo "MAKE=$MAKE" >> config.mak
@@ -173,7 +174,7 @@
 echo "gcinlibdir=\$(libdir)/gcin" >> config.mak
 echo "includedir=\$(DESTDIR)$includedir" >> config.mak
 
-echo "LDFLAGS=$GTKLDFLAGS" >> config.mak
+echo "LDFLAGS=$LDFLAGS" >> config.mak
 echo "GCIN_VERSION=$GCIN_VERSION" >> config.mak
 echo "GCIN_TABLE_DIR=\$(datadir_r)/gcin/table" >> config.mak
 echo "GCIN_OGG_DIR=\$(datadir_r)/gcin/ogg" >> config.mak
Index: gcin/qt-im/Makefile
===================================================================
--- gcin.orig/qt-im/Makefile	2008-12-06 14:28:01.608493456 +0800
+++ gcin/qt-im/Makefile	2008-12-06 14:29:25.788493516 +0800
@@ -10,7 +10,7 @@
 CXXFLAGS=$(OPTFLAGS) $(INCS) -Wall -D_REENTRANT -fPIC  -DQT_THREAD_SUPPORT -DQT_PLUGIN -DQT_SHARED -DQT_NO_DEBUG
 OBJS=qgcininputcontextplugin.o qgcininputcontextplugin_my.o qgcininputcontext.o qgcininputcontext_x11.o
 .SUFFIXES:	.c .cpp .a .so .E .h
-LDFLAGS=-L/usr/lib/$(QT)/$(LIB) -L/usr/X11R6/$(LIB) -L../im-client -lqt-mt -lmng -ljpeg -lpng -lz \
+LDFLAGS=-Wl,--as-needed -L/usr/lib/$(QT)/$(LIB) -L/usr/X11R6/$(LIB) -L../im-client -lqt-mt -lmng -ljpeg -lpng -lz \
 -lXrender -lXrandr  -lXft -lfontconfig -lgcin-im-client \
 -lXext -lX11 -lm -lSM -lICE -ldl -lpthread
 all:	libqgcin.so
